src/coloring/greedy_d1_coloring.jl

```diff
@@ -7,7 +7,7 @@ color using greedy approach. The number of colors
 used may be equal or greater than the chromatic
 number χ(G) of the graph.
 """
-function color_graph(G::VSafeGraph,alg::GreedyD1Color)
+function color_graph(G::VSafeGraph, alg::GreedyD1Color)
     V = nv(G)
     result = zeros(Int64, V)
     result[1] = 1
```

Comment:
below it seems you are only using the values 0 and 1 of available, you could replace it with a `BitVector`

Reply:
Done